src/Components/Input/Input.tsx

```diff
@@ -15,28 +31,48 @@ const Input: React.FC<InputProps> = ({ id, placeholder }) => {
     };
 
     const handleBlur = (event: React.FocusEvent<HTMLInputElement>) => {
-        if (!event.target.value) {
-            setErrorMessage("내용을 입력하세요.");
+        const value = event.target.value;
+        if (!value) {
+            setErrorMessage(placeholder);
+        } else if (id === "email" && !validateEmail(value)) {
+            setErrorMessage("올바른 이메일 주소가 아닙니다.");
+        } else if (id === "password" && !validatePassword(value)) {
+            setErrorMessage("비밀번호는 영문, 숫자 조합 8자 이상 입력해 주세요.");
         } else {
             setErrorMessage("");
         }
+
+        if (onBlur) {
+            onBlur(event);
+        }
     };
 
+    const inputType = type === "password" && passwordVisible ? "text" : type;
+
     return (
         <div className={styles.input_container}>
             <input
                 id={id}
-                className={`${styles.input} ${errorMessage ? styles.invalid : ""}`}
-                type={passwordVisible ? "text" : "password"}
+                className={`${styles.input} ${
+                    errorMessage || externalErrorMessage ? styles.invalid : ""
+                }`}
+                type={inputType}
                 placeholder={placeholder}
+                value={value}
+                onChange={onChange}
                 onBlur={handleBlur}
+                onKeyDown={onKeyDown}
             />
-            {errorMessage && <span className={styles.error_msg}>{errorMessage}</span>}
-            <button
-                className={`${styles.eye_button} ${passwordVisible ? styles.on : ""}`}
-                onClick={handleEyeToggle}
-                type='button'
-            ></button>
+            {(errorMessage || externalErrorMessage) && (
+                <span className={styles.error_msg}>{errorMessage || externalErrorMessage}</span>
+            )}
+            {type === "password" && (
+                <button
+                    className={`${styles.eye_button} ${passwordVisible ? styles.on : ""}`}
+                    onClick={handleEyeToggle}
+                    type='button'
+                ></button>
```

Comment:
내부에 자식 없으면 self closing 으로 많이 써요
```suggestion
                />
```